app/config/variables.php

```diff
@@ -420,12 +420,93 @@
+                'introduction' => '0.13.0',
+                'default' => '',
+                'required' => false,
+                'question' => '',
+            ],
+            [
+                'name' => '_APP_STORAGE_DEVICE_S3_REGION',
+                'description' => 'AWS S3 storage region. Required when storage adapter is set to S3. You can find your region info for your bucket from AWS console.',
+                'introduction' => '0.13.0',
+                'default' => 'us-eas-1',
+                'required' => false,
+                'question' => '',
+            ],
+            [
+                'name' => '_APP_STORAGE_DEVICE_S3_BUCKET',
+                'description' => 'AWS S3 storage bucket. Required when storage adapter is set to S3. You can create buckets in your AWS console.',
+                'introduction' => '0.13.0',
+                'default' => '',
+                'required' => false,
+                'question' => '',
+            ],
+            [
+                'name' => '_APP_STORAGE_DEVICE_DO_SPACES_ACCESS_KEY',
+                'description' => 'DigitalOcean spaces access key. Required when the storage adapter is set to DOSpaces. You can get your access key from your DigitalOcean console.',
+                'introduction' => '0.13.0',
+                'default' => '',
+                'required' => false,
+                'question' => '',
+            ],
+            [
+                'name' => '_APP_STORAGE_DEVICE_DO_SPACES_SECRET',
+                'description' => 'DigitalOcean spaces secret key. Required when the storage adapter is set to DOSpaces. You can get your secret key from your DigitalOcean console.',
+                'introduction' => '0.13.0',
+                'default' => '',
+                'required' => false,
+                'question' => '',
+            ],
+            [
+                'name' => '_APP_STORAGE_DEVICE_DO_SPACES_REGION',
+                'description' => 'DigitalOcean spaces region. Required when storage adapter is set to DOSpaces. You can find your region info for your space from DigitalOcean console.',
+                'introduction' => '0.13.0',
+                'default' => 'us-eas-1',
+                'required' => false,
+                'question' => '',
+            ],
+            [
+                'name' => '_APP_STORAGE_DEVICE_DO_SPACES_BUCKET',
+                'description' => 'DigitalOcean spaces bucket. Required when storage adapter is set to DOSpaces. You can create spaces in your DigitalOcean console.',
+                'introduction' => '0.13.0',
+                'default' => '',
+                'required' => false,
+                'question' => '',
+            ],
         ],
     ],
     [
         'category' => 'Functions',
         'description' => '',
         'variables' => [
+            [
+                'name' => '_APP_FUNCTIONS_DEPLOYMENT_LIMIT',
```

Comment:
```suggestion
                'name' => '_APP_FUNCTIONS_SIZE_LIMIT',
```

I feel `_APP_FUNCTIONS_SIZE_LIMIT` or `_APP_DEPLOYMENT_SIZE_LIMIT` conveys the meaning better.  LMK your thoughts

Reply:
_APP_FUNCTIONS because it's related to functions and in functions category, it's not a function's size limit but limit for DEPLOYMENT_SIZE. WDYT?